README.md

```diff
@@ -175,36 +179,16 @@ Creating and cloning a build:
       "container_name": "base",
       "config_file": "lxc.config",
       "init_timeout": "120s",
-      "template_parameters": [
-        "-d",
-        "ubuntu",
-        "-r",
-        "trusty",
-        "-a",
-        "amd64"
-      ],
-      "template_name": "ubuntu",
-      "cleanup_first": true
+      "rootfs": {
+        "archive": "/path/to/rootfs.tar.gz",
+        "config": "/path/to/lxc.config"
```

Comment:
Where are we going to get lxc.config in our emulator use case? Is that another artifact that we have to store? That doesn't change the code here, it's a good design, I'm just curious if you have any ideas.